controller/share_manager_controller.go

```diff
@@ -731,21 +733,25 @@ func (c *ShareManagerController) createShareManagerPod(sm *longhorn.ShareManager
 	mountOptions := pv.Spec.MountOptions
 
 	var cryptoKey string
+	var cryptoParams *crypto.EncryptParams
 	if volume.Spec.Encrypted {
 		secretRef := pv.Spec.CSI.NodePublishSecretRef
 		secret, err := c.ds.GetSecretRO(secretRef.Namespace, secretRef.Name)
 		if err != nil {
 			return nil, err
 		}
 
-		cryptoKey = string(secret.Data["CRYPTO_KEY_VALUE"])
+		cryptoKey = string(secret.Data[csi.CryptoKeyValue])
 		if len(cryptoKey) == 0 {
-			return nil, fmt.Errorf("missing CRYPTO_KEY_VALUE in secret for encrypted RWX volume %v", volume.Name)
+			return nil, fmt.Errorf("missing %v in secret for encrypted RWX volume %v", csi.CryptoKeyValue, volume.Name)
```

Comment:
Should not expose any credentials to log.

Reply:
`csi.CryptoKeyValue` is exactly the string `CRYPTO_KEY_VALUE`, I could changed backup if any concerns